test-suite/doc/imgs/DeploymentDoc.md

```diff
@@ -0,0 +1,81 @@
+# Deploying the 111-PEM-Adaptor test site to Azure/AWS
+
+## Contents
+1. [Azure](#Azure)
+2. [AWS](#AWS)
+
+## Azure
+### Pre-requisites
+- An Azure account (https://azure.microsoft.com/en-gb/)
+
+### Deployment Steps (via portal)
+1. To get started with deploying to Azure, the first thing you will need is an Azure account already set up. Navigate to the Azure Portal and click on “Storage Accounts”.
+2. Click on “Create” which can be found in the top left and go ahead and fill in the details (create your resource group and storage account name).
+    - For the “Redundancy” option, change to “Locally redundant storage” (LRS). Then go ahead and click “Review + Create”. ![image of creating a storage account](./azure/create-storage-account.png)
+3. Once your storage account has been created, go ahead and click into it. On the left of the screen will be a column, select “Static website” under the Data management section. 
+
+![image of select option static website](./azure/static-website-select.png)
+
```

Comment:
If you remove this empty line here and the line 16, then it'll render the next steps (line 19-20) nicely. Now they're rendered as a separate block of text (at least it looks this way in my IntelliJ's preview).